src/pageComponents/search/Results.jsx

```diff
@@ -69,8 +75,13 @@ class Results extends PureComponent {
     const { results } = this.props;
     return results.map((result, i) => {
       const { _index, _source: { title, url }, formattedDescription } = result;
+      const pathname = result._source.url;
       return (
-        <MediaWrapper key={ i } url={ url }>
+        <div key={ i }>
+        <Breadcrumbs path={ _index === 'guides' ?
```

Comment:
Indent `<Breadcrumbs ... />`

Reply:
Just for the furture, if we are breaking an attribute down over multiple lines, it should be done so like so:

```jsx
<Breadcrumbs
  path={
    _index === 'guides' ?
      pathname :
      breadcrumbTexts[_index]
  }
/> 
```

Reply:
@Bouncey Thanks, I didn't know that. Will make the change.